docs/_layouts/docs.html

```diff
@@ -3,193 +3,227 @@
+                </a>
+            </li>
+        </ul>
+
+        <h5>Application Deployment</h5>
+        <ul class="list-unstyled">
+            <li>
+                <a href="{{ site.baseurl }}/docs/application-basics.html">
+                    Application Basics
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/deployments.html">
+                    Application Deployments
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/command-line-flags.html">
+                    Command Line Flags
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/developing-vm.html">
+                    Run a Local Version of Marathon
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/native-docker-private-registry.html">
+                    Using a Private Docker Registry
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/artifact-store.html">
+                    Artifact Store
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/constraints.html">
+                    Constraints
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/event-bus.html">
+                    Event Bus
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/health-checks.html">
+                    Health Checks
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/ip-per-task.html">
+                    IP-per-task
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/task-environment-vars.html">
+                    Task Environment Variables
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/">
+                    Service Discovery and Load Balancing
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/ports.html">
+                    Ports
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/persistent-volumes.html">
+                    Stateful Applications Using Local Persistent Volumes
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/external-volumes.html">
+                    Stateful Applications Using Persistent External Volumes
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/recipes.html">
+                    Recipes
+                </a>
+            </li>
+        </ul>
+
+        <h5>Management</h5>
```

Comment:
It seems like "Administration" would be more accurate than configuration. For a few of these topics I hesitated about whether they would be more appropriate to "using" or "administering". If you see things misplaced, let me know!